PROTOCOL.md

```diff
@@ -603,6 +613,102 @@ Reader Version 2 | Respect [Column Mapping](#column-mapping)
 
 # Appendix
 
+## Deletion Vectors
```

Comment:
I think the deletion vector as a concept needs to be put much before Appendix. Appendix is great for details of the format etc. but top-level concepts like DV must be introduced earlier like Column Mapping. Please follow that structure.